aws-kendra-index/src/main/java/software/amazon/kendra/index/CreateHandler.java

```diff
@@ -176,13 +182,19 @@ private UpdateIndexResponse postCreate(
             final UpdateIndexRequest updateIndexRequest,
             final ProxyClient<KendraClient> proxyClient) {
         UpdateIndexResponse updateIndexResponse;
+
+        // Map UpdateIndex errors: https://code.amazon.com/packages/AWSKendraFrontendServiceModel/blobs/c901da56e4d60392385f5f1bce8e5e783157abc2/--/model/index_management_operations.xml#L24
+        // to cfn errors.
```

Comment:
I don't think we should reference an internal package. Can you please remove or change to reference public documentation https://docs.aws.amazon.com/kendra/latest/APIReference/API_UpdateIndex.html#API_UpdateIndex_Errors?

Reply:
+1

Reply:
ah,  looks like I didn't commit this update. Will send update.